### PR TITLE
docs: give a working Docker example

### PR DESCRIPTION
**Pull request by aykevl**

### CODE REVIEW

File: docs/docker.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ Using with Docker
 A docker container exists for easy access to the ``tinygo`` CLI.
 For example, to compile the blinky example from the root of the repository::
```
Comment (johanbrandhorst) [inline docs/docker.rst]: This line is now out of date

Comment (aykevl) [reply docs/docker.rst]: Good catch!

Comment (aykevl) [reply docs/docker.rst]: Agreed with the change? Then I will merge it.

**Comment by justinclift** [comment]
Should the main README.markdown add WebAssembly to the Supported Targets list too? :smile:

**Comment by aykevl** [comment]
@justinclift good point, added in 9393cdd5edab9d2378e1dd79fe2d03e9886e20ec.

**Pull request merged**